README.md

```diff
@@ -1,2 +1,39 @@
 # CodeacademyCourses
 A small django app to fetch all the completed courses on Codeacademy
+
+## Setup
+
+```sh
+$ https://github.com/Monal5031/CodeacademyCourses.git
+$ cd CodeacademyCourses
+```
+
+### For setting virtual environment if required
+```sh
+$ virtualenv venv
+```
+
+### For activating virtual environment in windows
+```sh
+$ venv/Scripts/activate
+```
+
+### For activating virtual environment in Ubuntu
+```sh
+$ source venv/bin/activate
+```
+
+After creating virtual environment 
+ 
+### Start
+```sh
+$ pip install -r requirements.txt
+$ python manage.py runserver
+```
+### To make migrations
+```sh
+$ python manage.py makemigrations
+$ python manage.py migrate
+```
+### Author :pencil:
+©️ [balwanishivam](https://github.com/balwanishivam) :tada:
```

Comment:
🤨

Reply:
Sorry i forgot to remove author i will make these changes

Reply:
Changes made